deployment/ansible/roles/cac-tripplanner.app/tasks/dev-test-dependencies.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Install dev/test python packages
+  pip: name={{ item.name }} version={{ item.version }}
+  with_items:
+    - { name: 'flake8', version: '2.3.0' }
```

Comment:
Yep, I saw that conversation; I think that having those two packages (ipython and django_extensions) available on production servers is a good idea because it can really help with debugging, but I also think we should separate out development-specific python libraries if there's a low chance we'll use them in production.